src/filters/TrimString.ts

```diff
@@ -0,0 +1,14 @@
+export default function(str: string, wordsCount: number = 15, omission: string = '...'): string {
+    if (typeof str !== 'string') {
+        return str
+    }
+
+    const words = str.trim().split(/\s+/)
+    if (words.length <= wordsCount) {
+        return str
+    }
+
+    const lastIncludedWord = words[wordsCount - 1]
```

Comment:
Will this work if there is only one word?

Reply:
Should be fine (unit tests would be handy for this). Bun wordsCount < 1 can break it, so I added a check for this, PR updated.